studio/lib/queries/page.ts

```diff
@@ -77,7 +77,7 @@ export const SLUG_QUERY = groq`
 `;
 
 export const SEO_SLUG_QUERY = groq`
-  *[_type == "pageBuilder" && slug.current == $slug][0]{
+  *[defined(seo) && slug.current == $slug][0]{
```

Comment:
how would this transfer for documents where we don't have or want seo? I'm assuming the default SEO would then be the fallback? Say for Legal documents

Reply:
perff